src/main/java/org/broadinstitute/dsm/route/CreateClinicalDummyKitRoute.java

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.dsm.route;
+
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.handlers.util.Result;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.db.KitType;
+import org.broadinstitute.dsm.statics.RequestParameter;
+import org.broadinstitute.dsm.util.DBUtil;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+import java.util.List;
+import java.util.Map;
+
+public class CreateClinicalDummyKitRoute implements Route {
+    private static final Logger logger = LoggerFactory.getLogger(CreateClinicalDummyKitRoute.class);
+    private static String CLINICAL_KIT_REALM = "CLINICAL_KIT_REALM";
+    private static int REALM;
+    private static final String USER_ID = "74";
+
+    @Override
+    public Object handle(Request request, Response response) throws Exception {
+        String kitLabel = request.params(RequestParameter.LABEL);
+        if (StringUtils.isBlank(kitLabel)) {
+            logger.warn("Got a create Clinical Kit request without a kit label!!");
+            return new Result(400, "Please include a kit label as a path parameter");
+        }
+        logger.info("Got a new Clinical Kit request with kit label " + kitLabel);
+        REALM = (int) DBUtil.getBookmark(CLINICAL_KIT_REALM);
+        KitType salivaKitType = null;
+        DDPInstance ddpInstance = DDPInstance.getDDPInstanceById(REALM);
+        if (ddpInstance != null) {
+            List<KitType> kitTypeIds = KitType.getKitTypes(ddpInstance.getName(), null);
+
+            for (KitType kitType : kitTypeIds) {
```

Comment:
salivaKitType = `kitTypeIds.stream.filter(kitType -> "SALIVA".equals(kitType.getName())).findFirst().orElseThrow()`, I think this approach is much more concise and protected from unexpected later NPEs.